### PR TITLE
Remove TestData from series-tests test_repr.py

### PR DESCRIPTION
**Pull request by SaturnFromTitan**
Part of #22550

* Replaced TestData usage in `pandas/tests/series/test_repr.py` with fixtures

- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by SaturnFromTitan** [comment]
I added a new empty_series fixture as it is mentioned in the issue description. I've seen more places where this can be used, so I hope this is fine with you

**Review by WillAyd — APPROVED** [review]
lgtm

File: pandas/tests/series/conftest.py [hunk 1]
```diff
@@ -31,3 +32,11 @@ def object_series():
     s = tm.makeObjectSeries()
     s.name = "objects"
     return s
+
```
Comment (jreback) [inline pandas/tests/series/conftest.py]: don't add these fixtures, just use Series()

**Review by jreback — CHANGES_REQUESTED** [review]

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks

**Pull request merged**